### ISSUE TITLE
Analyzing the low brightness emission

### ISSUE THREAD

**Issue by jpinedaf** [GBTAmmoniaSurvey/GAS#58]
This is to start the discussion of how to analyse the low brightness emission. A good starting point might be: Cappellari & Copin (2003) http://adsabs.harvard.edu/abs/2003MNRAS.342..345C

**Comment by jpinedaf** [comment]
some of the other useful papers are from the SAURON collaboration:
Emsellem et al. (2004) http://esoads.eso.org/abs/2004MNRAS.352..721E
Emsellem et al. (2007) http://esoads.eso.org/abs/2007MNRAS.379..401E http://mnras.oxfordjournals.org/content/379/2/401.full.pdf

**Comment by low-sky** [comment]
Yes!  This is exactly the approach I was talking about on the telecon a month ago.  It's going to require a good moment-1 estimate to get this correct.  From that, you can build the S/N maps you need.  To that end I was stewing over how to do better moments in the low S/N regime.  I started with just a windowed moment 1 but this can include a formalism to use weights given the hyperfine structure. [http://nbviewer.ipython.org/gist/low-sky/9feb28665cae25387dab](Here's my first dabbling).  I've wanted to test and see if it improves the estimate over the naive case.

**Comment by jpinedaf** [comment]
I see that this can be use fairly easily in complement with the code available [here](http://www-astro.physics.ox.ac.uk/~mxc/software/) for the Voronoi Tessellations. The code so far is in 2D, and stores everything in ASCII format... which might be a problem for our very large maps.

I can see us using this code to determine which pixels to combine, and shift them accordingly by the velocity estimate from the first moment.

Do you have some idea on how to store the Tessellations efficiently?

**Comment by low-sky** [comment]
The 2D aspect makes sense.  The application here will be to just 2D maps with a shift-and-stack.  This isn't an object identifier in 3D and isn't appropriate for that.  It would be good to bring it into our formalism.  Let me take a preliminary look at this tonight when I'm observing (provided it's not too windy).
